src/main/java/io/cdap/plugin/salesforce/plugin/sink/batch/SalesforceSinkConfig.java

```diff
@@ -418,16 +418,17 @@ private SObjectsDescribeResult getSObjectDescribeResult(FailureCollector collect
    *
    * @param schema input schema to check
    */
-  private void validateInputSchema(Schema schema) {
-    if (connection != null) {
-      AuthenticatorCredentials authenticatorCredentials = connection.getAuthenticatorCredentials();
-      try {
-        SObjectDescriptor sObjectDescriptor = SObjectDescriptor.fromName(sObject, authenticatorCredentials);
-        Schema sObjectActualSchema = SalesforceSchemaUtil.getSchema(authenticatorCredentials, sObjectDescriptor);
-        SalesforceSchemaUtil.checkCompatibility(sObjectActualSchema, schema, false);
-      } catch (ConnectionException e) {
-        throw new InvalidStageException("There was issue communicating with Salesforce", e);
-      }
+  private void validateInputSchema(Schema schema, FailureCollector collector) {
+    AuthenticatorCredentials authenticatorCredentials = connection.getAuthenticatorCredentials();
```

Comment:
Is there a check for macro needed here? Or is it checked somewhere else?

Reply:
it is already checked before, if connection properties are macro or having wrong values, that has been validated in validate method in starting.